dist/infragram.js

```diff
@@ -548,9 +548,13 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
         lnk.download = new Date().toISOString().replace(/:/g, "_") + "." + format; // create a "fake" click-event to trigger the download
 
         if (document.createEvent) {
-          event = document.createEvent("MouseEvents");
-          event.initMouseEvent("click", true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
-          lnk.dispatchEvent(event);
+          const mouseEvent = getMouseEvent();
```

Comment:
Hi @Somya-Singhal sorry for my slow reply as we have a lot of activity from newcomers right now. Basically, these additions can also be made on this line - 

https://github.com/Somya-Singhal/infragram/blob/19e01e9b90d75bf3b42552f8450998fde28e57b3/src/io/file.js#L18-L25

And what will happen is you can then run the command `grunt build` and it will compile your changes and all other files in /src/ into the /dist/ folder. The dist folder is actually auto-generated, we really only edit the /src/ files. So if you can make your changes in /src/ and compile, they'll show up in /dist/. Does that make sense? 

Thank you so much!

Reply:
Ok @jywarren, I will try this and inform you soon.

Reply:
Hello @jywarren, I made changes in /src/file then ran grunt build.After that the following changes were made:
1)In dist/infragram.js

<img width="724" alt="image" src="https://user-images.githubusercontent.com/88088520/163053255-20fb6f49-bb3c-40e9-89c8-e3cf4c1ed62c.png">

<img width="747" alt="image" src="https://user-images.githubusercontent.com/88088520/163053328-1e252c06-b445-4f2c-ad02-9969deea94b7.png">

<img width="688" alt="image" src="https://user-images.githubusercontent.com/88088520/163053388-04f7f161-4b96-4e02-8b47-54544e492e07.png">

<img width="719" alt="image" src="https://user-images.githubusercontent.com/88088520/163053442-e20a73a0-2e66-44c9-be83-d106152da981.png">

<img width="754" alt="image" src="https://user-images.githubusercontent.com/88088520/163053505-15c7273d-8733-41da-8c48-b8ca1d63bb63.png">

<img width="739" alt="image" src="https://user-images.githubusercontent.com/88088520/163053560-aa9c87a5-ffe5-422a-9bca-058b68759f7b.png">

<img width="808" alt="image" src="https://user-images.githubusercontent.com/88088520/163053631-ee34de1b-d0d4-4eff-b40f-c4e3a3db536b.png">

<img width="741" alt="image" src="https://user-images.githubusercontent.com/88088520/163053698-3135f896-010a-4369-8b7b-cdf90bef2e2c.png">


2)In dist/infragram.js.map
<img width="757" alt="image" src="https://user-images.githubusercontent.com/88088520/163053846-f806629c-24ca-4790-8061-02b8b9118cb8.png">

3) In dist/infragram.min.js
<img width="758" alt="image" src="https://user-images.githubusercontent.com/88088520/163053926-e855472b-6ddf-42fe-a3ee-0aa0eaa86a6f.png">

Had I done something wrong?🙂

Reply:
That looks right! Can you now `git add` the file in src, and push it to the same remote branch to add it to this PR? Thank you!

Reply:
Thanks @jywarren for helping me. I linked a PR #365, if any further changes are required please do let me know.

Reply:
Can you please review it and do let me know it I made any mistake?